src/components/legend/defaults.js

```diff
@@ -33,9 +33,9 @@ function groupDefaults(legendId, layoutIn, layoutOut, fullData) {
     };
 
     var globalFont = layoutOut.font || {};
-    var grouptitlefont = Lib.coerceFont(coerce, 'grouptitlefont', Lib.extendFlat({}, globalFont, {
+    var grouptitlefont = Lib.coerceFont(coerce, 'grouptitlefont', globalFont, { overrideDflt: {
         size: Math.round(globalFont.size * 1.1)
```

Comment:
why '1.1' ?

Reply:
Good question. It's not related to changes of this PR so I need to look into the previous pull requests.
Possibly we wanted to have a larger size for the title and `1.2` was too large.

Reply:
`1.2` was used for main legend title here: 
https://github.com/plotly/plotly.js/blob/c50363479539e943b1682c83c734c1ed4a7f8704/src/lib/index.js#L1423-L1425
So for group titles a value used between 1.2 and 1.